cpp/inc/bond/core/reflection.h

```diff
@@ -383,13 +383,32 @@ remove_bonded
 };
 
 
-template <typename T> struct
-remove_bonded<bonded<T> >
+template <typename T, typename Reader> struct
+remove_bonded<bonded<T, Reader> >
 {
     typedef typename remove_bonded<T>::type type;
 };
 
 
+template <typename T> struct
+remove_bonded_value
+{
+    typedef T type;
```

Comment:
why not using?

Reply:
To be in-sync with existing old style in this header.